target/linux/ar71xx/files/arch/mips/ath79/machtypes.h

```diff
@@ -37,6 +37,7 @@ enum ath79_mach_type {
 	ATH79_MACH_AP143,			/* Atheros AP143 reference board */
 	ATH79_MACH_AP147_010,			/* Atheros AP147-010 reference board */
 	ATH79_MACH_AP152,			/* Atheros AP152 reference board */
+	ATH79_MACH_AP531B0,			/* Rockeetech AP531B0 board */
```

Comment:
You have also changed file attributes (`100644 → 100755`) which is **wrong** - please, fix it.